hr_expense_petty_cash/models/hr_expense_sheet.py

```diff
@@ -74,3 +76,11 @@ def _check_petty_cash_amount(self):
                             company_currency.symbol,
                         )
                     )
+
+    @api.onchange("employee_id")
```

Comment:
If you have new features, please split this PR to 2 commit
- migration
- implement

but I suggest create new PR for IMP only